release-notes/7.0/supported-os.md

```diff
@@ -28,13 +28,13 @@ OS                                    | Version                 | Architectures
 OS                                    | Version               | Architectures     | Lifecycle
 --------------------------------------|-----------------------|-------------------|----------
 [Alpine Linux][Alpine]                | 3.15+                 | x64, Arm64, Arm32 | [Alpine][Alpine-lifecycle]
-[CentOS Linux][CentOS]                | 7                     | x64               | [CentOS][CentOS-lifecycle]
-[CentOS Stream Linux][CentOS]         | 8                     | x64               | [CentOS][CentOS-lifecycle]
+[CentOS][CentOS]                      | 7                     | x64               | [CentOS][CentOS-lifecycle]
+[CentOS Stream][CentOS]               | 8+                    | x64               | [CentOS][CentOS-lifecycle]
 [Debian][Debian]                      | 10+                   | x64, Arm64, Arm32 | [Debian][Debian-lifecycle]
 [Fedora][Fedora]                      | 36+                   | x64               | [Fedora][Fedora-lifecycle]
 [openSUSE][OpenSUSE]                  | 15+                   | x64               | [OpenSUSE][OpenSUSE-lifecycle]
```

Comment:
openSUSE lists 15+ but technically it's 15.4+ which isn't EOL. Not sure if this is intentional to signalize Microsoft support while OS is EOL.

Reply:
We try not to chase minor versions as much as possible but it may make sense in this case.